interface/utils.h

```diff
@@ -36,7 +42,11 @@ std::map<int,std::vector<float>> nsig_years = {
   {2018, {500, 981, 821, 1608, 0.002, 3018, 0.003, 1836}},
 };
 
-
+std::map<int,std::vector<float>> constr_width_fac = {
```

Comment:
can you explain how these numbers are derived?

Reply:
From the results of the MC mass fits, the ratios between the relative uncertainty in the signal q2 bins and in the Jpsi bin are computed only for the parameters in common between the two mass shapes, and excluding the mean parameters. The value reported here is the simple average between these values computed for each bin/tag/year configuration